packages/flutter/lib/src/material/switch_list_tile.dart

```diff
@@ -375,6 +402,73 @@ class SwitchListTile extends StatelessWidget {
   /// The color for the tile's [Material] when a pointer is hovering over it.
   final Color? hoverColor;
 
+  /// An optional error callback for errors emitted when loading
```

Comment:
Strongly consider using macros to avoid duplicating docs

Reply:
Added macros for some of them. There are also some APIs like thumbColor, trackColor and etc., their focus state cannot be applied because the ListTile will be focused when SwitchListTile requests focus, so I just didn't use template for them.